ServiceWebsite/ServiceWebsite.Services/HearingsService.cs

```diff
@@ -43,7 +43,40 @@ private static bool HearingContainsParticipant(HearingDetailsResponse response,
         {
             return response.Participants.Any(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
         }
-        
+
+        public async Task<Guid> GetParticipantId(string username, Guid id)
+        {
+            try
+            {
+                var hearingResponse = await _bookingsApiClient.GetHearingDetailsByIdAsync(id);
+                var participant = hearingResponse.Participants.First(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
+
+                if (participant == null )
+                {
+                    throw new UnauthorizedAccessException($"User is not participant of hearing: {id}");
+                }
+
+                var participantId = participant.Id ?? Guid.Empty;
```

Comment:
Since the participant id is retrieved from the service it will never ever be empty. So you can simply return it straight away